DQMServices/Components/scripts/apipopularity.sh

```diff
@@ -0,0 +1,17 @@
+(sqlite3 $1 | column -t -s'|') <<SQL
```

Comment:
These files end up in everyones `$PATH`. Ok, they don't seem to have their execution bit set, and they're missing `#!` so one can't really execute them (without explicitly passing them to a shell), but to me most of the scripts have rather generic names (like `csv_to_sqlite.sh` or all the `*dxr*` ones) leading to risks of name collisions.

Reply:
Yep, I am aware. Any better place to hide such things without removing them entirely?

I guess most of them can be deleted in the end, but a few could remain useful.